core/src/main/java/com/novoda/noplayer/player/NoPlayerCreator.java

```diff
@@ -0,0 +1,53 @@
+package com.novoda.noplayer.player;
+
+import android.content.Context;
+
+import com.novoda.noplayer.Player;
+import com.novoda.noplayer.drm.DrmHandler;
+import com.novoda.noplayer.drm.DrmType;
+import com.novoda.noplayer.exoplayer.NoPlayerExoPlayerCreator;
+import com.novoda.noplayer.exoplayer.drm.DrmSessionCreator;
+import com.novoda.noplayer.exoplayer.drm.DrmSessionCreatorFactory;
+import com.novoda.noplayer.mediaplayer.NoPlayerMediaPlayerCreator;
+
+class NoPlayerCreator {
```

Comment:
This is the old `PlayerFactory` minus a few wrapper methods, we only expose the one method create and it is used only in the `PlayerBuilder`.